config.toml

```diff
@@ -33,6 +33,15 @@ sqlite_datasource = "./aggregator.db"
 log_sql_queries = true
 type = "sql"
 
+[dvo_recommendations_storage]
+db_driver = "sqlite3"
+sqlite_datasource = "./dvo.db"
```

Comment:
IMHO I don't think it makes much sense to support sqlite. It isn't supported at all for the "old" storage. In fact you can't use it as migrations will fail

Reply:
removed